src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -54,7 +54,7 @@
      * We assume that each recently added participant will contribute this much
      * to the bridge's load
      */
-    private static final Double AVG_PARTICIPANT_LOAD = config.averageParticipantLoad();
+    private static final Double AVG_PARTICIPANT_STRESS = config.averageParticipantStress();
```

Comment:
Do we need the constant in addition to the value in config?

Reply:
Good point, we probably don't.  I can get rid of it.

Reply:
Asks the person who did the same on the line above...